### DISCUSSION TITLE
Automation via node or others

### DISCUSSION THREAD

**Discussion by FireBird22** [dyc3/steamguard-cli#412] [Q&A] [answered]
Hey,
I was trying to automate the setup process via nodeJS, I've tried to spawn a shell window and exec the ./steamguard.exe setup, everything works fine until it just randomly stops accepting input from the stdin stream, is there a way to automate this? or there is no support for it?

**Comment by dyc3** [comment, marked as answer]
steamguard-cli is not really meant to be used this way. You'd probably have a better experience using these libraries instead: 
- https://github.com/DoctorMcKay/node-steam-user
- https://github.com/DoctorMcKay/node-steam-session
